### DISCUSSION TITLE
HELP: .next folder in development mode storing all API call server responses

### DISCUSSION THREAD

**Discussion by nathanblon** [vercel/next.js#64140] [Help]
### Summary

Feel like I'm going crazy but every time I PUT and change something in my database, on reload in "npm run dev" mode the .next folder returns only the first ever response given, so not the reflected changes. I have to delete the .next folder and restart the development server to see the changes reflected. Why would it ever do that and how can I change it so it doesn't?

   const response = await fetch("/api/vercelkv/expos/get-all", {
          cache: "no-store",
        });
I tried cache "no-store" and other things to no avail.

**Comment by MarouaneBenbetka** [comment]
May be there was no change in the database or you nextjs project is linked to an old database .
